.circleci/config.yml

```diff
@@ -213,42 +253,50 @@ workflows:
             branches:
               only: master
 
-  on_tag_build_test_deploy:
+  on_tag_build_release_publish:
     jobs:
       - build:
-          filters:  # required since `release` has tag filters AND requires `build`
+          filters:
             tags:
               only: /release/
             branches:
               ignore: /.*/
       - test:
           requires:
             - build
-          filters:  # required since `release` has tag filters AND requires `test`
+          filters:
             tags:
               only: /release/
             branches:
               ignore: /.*/
-      - release:
+      - ghpages:
           requires:
-            - test
-          filters:  # job runs on tag 'release' for no branches
+            - build
+          filters:
             tags:
               only: /release/
             branches:
               ignore: /.*/
-      - ghpages:
+      - release-to-github:
           requires:
-            - test
-          filters:  # job runs on tag 'release' for no branches
+            - build
```

Comment:
won't you prefer the release process to fail if the tests fail?

Reply:
yes, you are right.

I change the workflow required relation, build->test->other steps